mock/docs/site-defaults.cfg

```diff
@@ -485,3 +485,6 @@
 # Fedora or EL systems work with that.  But some destination chroots can have
 # different configuration, and copying the bundle helps.
 #config_opts['ssl_ca_bundle_path'] = None
+
+# user_agent string to identify HTTP request to repositories
+# config_opts['user_agent'] = "mock (Fedora {{ releasever }}; mock; {{ target_arch }})"
```

Comment:
This needs to be synced with the real value.

Reply:
Ping @xsuchy at least about this.

Reply:
It is just a leftover comment, the code now mentiones `root`, which is `<distro>-<release>-<arch>` triplet.